client/src/modules/CrossCheck/components/SubmittedStatus.tsx

```diff
@@ -16,7 +16,8 @@ export function SubmittedStatus(props: Props) {
 
   if (!solution) {
     const deadlinePassedMessage = 'Submission deadline has already passed';
-    const message = `You haven't submitted solution. ${deadlinePassed ? deadlinePassedMessage : ''}`;
+    const tipMessage = 'Try to submit your solution as early as possible';
+    const message = `You haven't submitted solution. ${deadlinePassed ? deadlinePassedMessage : tipMessage}`;
```

Comment:
As far as I remember, we cannot submit after the deadline, so it seems odd to me to display such a text if submission is not possible.

Reply:
my bad